tests/dummy/app/components/contrib-manager/component.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import layout from './template';
+
+import permissions from 'ember-osf/const/permissions';
+
+export default Ember.Component.extend({
+    READ: permissions.READ,
+    WRITE: permissions.WRITE,
+    ADMIN: permissions.ADMIN,
+    layout: layout,
+    permissionChanges: {},
+    bibliographicChanges: {},
+    actions: {
+        addContributor(userId, permission, isBibliographic) {
+            this.sendAction('addContributor', userId, permission, isBibliographic);
+        },
+        removeContributor(contrib) {
+            this.sendAction('removeContributor', contrib);
+        },
+        permissionChange(contributor, permission) {
+            this.set(`permissionChanges.${contributor.id}`, permission.toLowerCase());
+        },
+        bibliographicChange(contributor, isBibliographic) {
+            this.set(`bibliographicChanges.${contributor.id}`, isBibliographic);
+        },
+        updateContributors() {
+            this.attrs.editContributors(
```

Comment:
`sendAction` doesn't return a value (https://github.com/emberjs/ember.js/blob/v2.6.0/packages/ember-views/lib/components/component.js#L283), whereas calling the method directly from attrs allows us to respond to create actions that return values.